### PR TITLE
test that shows how to create multiple tokens in setup actions

### PR DESCRIPTION
**Pull request by oveddan**
Shows how multiple tokens can be published in setup actions.  Only works for new contracts

### CODE REVIEW

**Comment by iainnash** [comment]
Would be great to rebase and merge.

**Comment by iainnash** [comment]
@oveddan ping?

**Comment by oveddan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #106** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/106" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/106?utm_source=stack-comment).

**Review by iainnash — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -42,12 +44,17 @@
...
     "prettier-plugin-solidity": "^1.1.3",
     "tsup": "^6.7.0",
-    "typescript": "^5.0.4"
-  }
+    "typescript": "^5.0.4",
+    "viem": "^1.6.0",
+    "vite": "^4.1.4",
+    "vitest": "~0.30.1"
+  },
+  "packageManager": "pnpm@7.9.4"
```
Comment (iainnash) [inline package.json:59]: Can we use pnpm here?

Comment (oveddan) [reply package.json:59]: that shouldn't have been added...will remove

Comment (oveddan) [reply package.json]: removed

**Pull request merged**